pkg/datapath/orchistrator/cells.go

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package orchistrator
```

Comment:
s/orchistrator/orchestrator/ (double-check my spelling too)

Should this be in `pkg/datapath/linux/orchestrator`?